codeHF/dpl-config_run5_hf.json

```diff
@@ -429,6 +469,46 @@
             ]
         }
     },
+    "hf-xicc-topkpipi-candidate-selector": {
```

Comment:
As commented in the original PR, this device name seems wrong.
The task name `HfXiccToPKPiPiCandidateSelector` should translate into the device name `hf-xicc-to-p-k-pi-pi-candidate-selector`. Please double-check it in the O2 stdout log.

Reply:
I checked the log, and it turns out the device name is `hf-xicc-to-p-k-pi-pi-candidate-selector`, as you mentioned. 
Is there any chance to change the device name myself? since I want to make it consistent with another selector name in config.

Reply:
The device name would have to be changed in the O2Physics task using `TaskName` (as can be seen in other tasks).